__tests__/db.test.js

```diff
@@ -210,3 +210,50 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH 200: responds with the updated article", () => {
+    const updatedArticle = { inc_votes: 20 };
+    const expectedArticle = {
+      article_id: 5,
+      title: "UNCOVERED: catspiracy to bring down democracy",
+      topic: "cats",
+      author: "rogersop",
+      body: "Bastet walks amongst us, and the cats are taking arms!",
+      created_at: "2020-08-03T13:14:00.000Z",
+      votes: 20,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .patch("/api/articles/5")
+      .send(updatedArticle)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(expectedArticle);
+      });
+  });
+  test(`PATCH 400: responds with error when patch request is missing required data`, () => {
```

Comment:
This is okay as 400 now, but in case you want to patch other properties, this test will fail. So it could be a 200 returning the unchanged article. It's up to you to decide.